src/util/identitiesUtils.js

```diff
@@ -123,9 +154,9 @@ export const unlockIdentitySeed = async (pin, identity) => {
 	return phrase;
 };
 
-export const getAvailableNetworkKeys = identity => {
-	const accountIdsList = Array.from(identity.accountIds.values());
-	const networkKeysSet = accountIdsList.reduce((networksSet, path) => {
+export const getExistedNetworkKeys = identity => {
```

Comment:
question: what does this mean? like legacy network keys?

Reply:
For example under the seed, if it has ropsten account, substrate account, kovan account, it will return a list of networkKey of ropsten, substrate, kovan.